save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveConfigReader.kt

```diff
@@ -74,10 +74,10 @@ fun createConfigFromArgs(args: Array<String>): SaveProperties {
     val testFiles = configFromCli.testFiles
     if (!testFiles.isNullOrEmpty() && !fs.exists(testFiles[0].toPath())) {
         // FixMe: get(0) to [0] after https://github.com/cqfn/diKTat/issues/1047
-        errorAndExitNotValidDir(testFiles!!.get(0).toPath())
+        errorAndExitNotValidDir(testFiles.get(0).toPath())
     }
     // FixMe: get(0) to [0] after https://github.com/cqfn/diKTat/issues/1047
```

Comment:
FixMe is actual, because of opened https://github.com/cqfn/diKTat/pull/1050

I just removed redundant `!!` access, since in these cases we already checked `testFiles` for nullability by `!testFiles.isNullOrEmpty()` and there were warning about it by compiler

Reply:
Yes, so you can now change `!!.get(0)` to simply `[0]`, even if the issue is still opened

Reply:
Really, didn't notice